Sample/Vungle Sample App/BannerList.m

```diff
@@ -0,0 +1,139 @@
+//
+//  BannerList.m
+//  Vungle Sample App
+//
+//  Created by John Mai on 2/6/20.
+//  Copyright © 2020 Jordyn Chuhaloff. All rights reserved.
+//
+
+#import "BannerList.h"
+#import "Constants.h"
+#import "DisplayTableViewCell.h"
+
+@interface BannerList () {
+    CGFloat screenHeight;
+    CGFloat screenWidth;
+}
+@property (strong, nonatomic) IBOutlet UIView *view;
+@property const int totalSize;
+@property const int AD_PER_ITEM;
+@property NSMutableArray* adArr;
+@property (nonatomic, strong) VungleSDK *sdk;
+@end
+
+@implementation BannerList
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.adArr = [[NSMutableArray alloc] init];
+    self.totalSize = 40;
+    self.AD_PER_ITEM = 4;
+    [self startVungle];
+    [self.view addSubview:self.tableView];
+    self.tableView.delegate = self;
+    self.tableView.dataSource = self;
+}
+-(void)viewDidAppear:(BOOL)animated {
+    [self.tableView reloadData];
+}
+-(NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    return 1;
+}
+- (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath {
+    
+    CGFloat height = 50.0;
+    if (indexPath.row == (self.totalSize - 4)) {
+        height = 250.0;
```

Comment:
Here, are you hard coding the height of a specific row for an MREC ad type?

Also for each - you're not allowing any padding at the top / bottom of the cell - is this intended?